afew/commands.py

```diff
@@ -91,13 +89,14 @@
     help='arguments for notmuch new (in move mode)'
 )
 
+
 def main():
-    if sys.version_info < (3,6):
+    if sys.version_info < (3, 6):
         sys.exit("Python 3.6 or later is required.")
 
     args = parser.parse_args()
 
-    no_actions = len(filter_compat(None, (
+    no_actions = len(filter(None, (
```

Comment:
This doesn't work since Python 3 filter objects do not have a length, so this will therefore raise.  Simplest solution would be to change it to `len(list(filter(...)))`

> TypeError: object of type 'filter' has no len()

Reply:
fixed in #255.